hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java

```diff
@@ -76,17 +78,16 @@ public static String getTestMethodName() {
     }
 
     /**
-     * Creates a BlockJUnit4ClassRunner to run {@code klass}
+     * Creates a BlockJUnit4ClassRunner to run {@code classType}
      *
      * @throws org.junit.runners.model.InitializationError if the test class is malformed.
      */
-    public AbstractHazelcastClassRunner(Class<?> klass) throws InitializationError {
-        super(klass);
```

Comment:
I just wanted to give the names a unified and more English name. I think I saw `classType` in the JDK itself, but I may be wrong. I picked `clazz` for now.